src/app/pages/send-crypto-asset/forms/stx-sip10/stacks-fungible-token-send-form.tsx

```diff
@@ -32,10 +34,12 @@ export function StacksFungibleTokenSendForm({ symbol }: StacksFungibleTokenSendF
     pullContractIdFromIdentity(contractId)
   );
   const ftAmountSchema = useFungibleTokenAmountSchema(contractId);
+  const txFees = useStacksFtFees();
 
   const initialValues = createDefaultInitialFormValues({
     symbol: '',
-    fee: null,
+    fee: 0,
+    feeType: 0,
```

Comment:
fee type 🤔

Reply:
This was added for analytics by Beguene I believe? I'll look if still need.